active_model_serializers.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.1'
 
-  rails_versions = ['>= 4.1', '< 6.1']
+  rails_versions = ['>= 4.1', '<= 6.1']
   spec.add_runtime_dependency 'activemodel', rails_versions
```

Comment:
This would exclude 6.1.1 no?

Reply:
yes.
```rb
  rails_versions = ['>= 4.1', '< 6.2']
```
might be ideal.

Reply:
or even may be `< 7.0` if we like to make it a bit loose :)

Reply:
Changed to 6.2. 7 would be too far fetched IMO. 😅

Reply:
@bf4 can we create a 0.10.11 release with these changes?